app/views/admin/users.html.erb

```diff
@@ -22,6 +22,15 @@
         <td><%= user.is_admin ? "Yes" : "No" %></td>
         <td><%= user.is_code_admin ? "Yes" : "No" %></td>
         <td><%= link_to "Feedback", admin_user_feedback_path(:user_id => user.id) %></td>
+        <% if user_signed_in? && current_user.is_admin %>
+          <td>
+            <% unless user.is_code_admin %>
+              <%= link_to "Grant code access", url_for(:controller => :admin, :action => :promote_code_admin, :id => user.id), :method => :post, :class => "text-danger" %>
```

Comment:
It'd probably be a decent idea to put a `:confirm` in here, with the name of who you're granting access to.